src/datanode/src/instance.rs

```diff
@@ -126,24 +119,8 @@ impl Instance {
                 let _meta_client = meta_client.context(IncorrectInternalStateSnafu {
                     state: "meta client is not provided when building heartbeat task",
                 })?;
-                let region_alive_keepers =
-                    region_alive_keepers.context(IncorrectInternalStateSnafu {
-                        state: "region_alive_keepers is not provided when building heartbeat task",
-                    })?;
-                let _handlers_executor = HandlerGroupExecutor::new(vec![
-                    Arc::new(ParseMailboxMessageHandler),
-                    Arc::new(OpenRegionHandler::new(
-                        catalog_manager.clone(),
-                        engine_manager.clone(),
-                        region_alive_keepers.clone(),
-                    )),
-                    Arc::new(CloseRegionHandler::new(
```

Comment:
Can we delete these (OpenRegionHandler, CloseRegionHandler)? It seems that region failover needs these.

Reply:
cc @MichaelScofield

Reply:
The new implementation is https://github.com/GreptimeTeam/greptimedb/blob/a297fefc708b5675d734da5772f692f99b887ca6/src/datanode/src/heartbeat/handler.rs